app/gui/src/project-view/stores/suggestionDatabase/entry.ts

```diff
@@ -192,8 +192,10 @@ const DOCUMENTATION_ROOT = 'https://help.enso.org/docs/api'
 export function suggestionDocumentationUrl(entry: SuggestionEntry): string | undefined {
   if (entry.kind !== SuggestionKind.Method && entry.kind !== SuggestionKind.Function) return
   const { project, path } = entry.definitionPath
-  if (!project?.startsWith('Standard.') || !path) return
-  return [DOCUMENTATION_ROOT, project, ...qnSegments(path)].join('/')
+  const definedPath = entry.definedIn.path + '.'
+  const usedPath = definedPath ? path?.replace(definedPath, definedPath?.replaceAll('.', '/')) : path
+  const concatenatedPath = `${DOCUMENTATION_ROOT}/${project}/${usedPath}`
```

Comment:
If I read this correctly, "definedPath" will never be falsy, as we always add `'.'` to it, right?

And as I understand, we send to the module's page, but there's no jump to a concrete definition.

Also, I'd prefer the old `join` implementation instead of regex, so we will have one place with regex for qualified names.

Reply:
I adjusted to using more of the old approach.
I'm not sure if I'm convinced that calling segments to split on `.` and then join on `/` is simpler, but I have reverted to this.